src/plugins/intel_gpu/tests/unit/test_cases/eltwise_gpu_test.cpp

```diff
@@ -3439,6 +3439,65 @@ TEST(eltwise_gpu_f32, broadcast_test_dim3_dim4) {
     }
 }
 
+TEST(eltwise_gpu_f32, broadcast_test_dim3_dim4_new_shape_infer_false) {
+    auto& engine = get_test_engine();
+
+    ov::Shape in2_shape = {1, 4, 1, 1};
+    auto input2 = engine.allocate_memory({ ov::PartialShape(in2_shape), data_types::f32, format::bfyx });
+
+    std::vector<float> const_input = {
+        1.f,   0.f, 5.f, 1.5f,
+        2.f,   0.f, 6.f, 5.2f,
+        3.f,  0.5f, 7.f, 12.f,
+        4.f, -0.5f, 8.f,  8.f
+    };
+
+    set_values(input2, {
+        0.5f,   2.5f,  0.5f,  2.5f
+    });
+
+    float answers[16] = {
+        1.5, 0.5, 7.5, 4,
+        2.5, 0.5, 8.5, 7.7,
+        3.5, 1, 9.5, 14.5,
+        4.5, 0, 10.5, 10.5
+    };
+
+    ExecutionConfig config = get_test_default_config(engine);
+    config.set_property(ov::intel_gpu::allow_new_shape_infer(false));
+
+    // in1:dim3, int2:dim4
+    {
+        ov::Shape in1_shape = {2, 4, 2};
```

Comment:
I don't think that input1_shape = {2, 4, 2}, input2_shape = {1, 4, 1, 1} is supported by `NUMPY` rule. Please refer https://docs.openvino.ai/2024/documentation/openvino-ir-format/operation-sets/broadcast-rules.html